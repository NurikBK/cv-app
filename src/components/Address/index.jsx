import React from 'react';
import { StyledAddress } from '../styles/Address.styled';

const Address = () => {
  return (
    <StyledAddress className="address">
      <dl>
        <dt></dt>
        <dd>
          <a href="tel:+500 342 242">500 342 242</a>
        </dd>
      </dl>
      <dl>
        <dt></dt>
        <dd>
          <a href="mailto:office@kamsolutions.pl">office@kamsolutions.pl</a>
        </dd>
      </dl>
      <dl>
        <dt></dt>
        <dd>
          <strong>Twitter</strong>
          <br />{' '}
          <a href="https://twitter.com/wordpress">
            https://twitter.com/wordpress
          </a>
        </dd>
      </dl>
      <dl>
        <dt></dt>
        <dd>
          <strong>Facebook</strong>
          <br />{' '}
          <a href="https://www.facebook.com/facebook">
            https://www.facebook.com/facebook
          </a>
        </dd>
      </dl>
      <dl>
        <dt></dt>
        <dd>
          <strong>Skype</strong>
          <br /> <a href="skype:kamsolutions.pl">kamsolutions.pl</a>
        </dd>
      </dl>
    </StyledAddress>
  );
};

export default Address;
