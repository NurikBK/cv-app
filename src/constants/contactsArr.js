import { faPhone, faEnvelope } from '@fortawesome/free-solid-svg-icons';
import {
  faTwitter,
  faFacebookF,
  faSkype,
} from '@fortawesome/free-brands-svg-icons';

export const contacts = [
  {
    icon: faPhone,
    title: '+7 708 693 23 41',
    info: '',
    link: 'tel:+7 708 693 23 41',
  },
  {
    icon: faEnvelope,
    title: 'bizhannk@gmail.com',
    info: '',
    link: 'mailto:bizhannk@gmail.com',
  },
  {
    icon: faTwitter,
    title: 'Twitter',
    info: '@Nurik',
    link: 'https://twitter.com/wordpress',
  },
  {
    icon: faFacebookF,
    title: 'Facebook',
    info: 'Nurzhan Bizhan',
    link: 'https://www.facebook.com/facebook',
  },
  {
    icon: faSkype,
    title: 'Skype',
    info: 'Nurik',
    link: 'skype:nurik',
  },
];
