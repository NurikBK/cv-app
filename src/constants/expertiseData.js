export const expertiseData = [
  {
    date: '2013-2023',
    info: {
      company: 'Asia Gas Pipeline',
      job: 'Front-end developer / php programmer',
      description: `Translate written and spoken content accurately and efficiently from Chinese to English, English to Chinese, Russian to English, and English to Russian
        Proofread and edit translations to ensure accuracy and clarity of the content
        Research industry-specific terminology to ensure that translations are accurate and appropriate
        Use translation software and tools to assist in the translation process
        Collaborate with other translators or editors to ensure consistency and accuracy across translations
        Maintain confidentiality of translated content and client information
        Communicate with clients to understand their needs and preferences and ensure that the translated content meets their expectations
        Meet project deadlines and manage time effectively`,
    },
  },
  {
    date: '2023 feb - 2023 may',
    info: {
      company: 'Bank Center Credit',
      job: 'Junior Web developer',
      description: `Creating and managing content for social media platforms such as Facebook, Twitter, and Instagram.

        Writing blog posts and articles for the company website.
        
        Conducting research and gathering information on industry-related topics to inform content creation.
        
        Collaborating with the marketing team to create and execute social media and content marketing strategies.
        
        Managing social media accounts by monitoring engagement and responding to user comments and messages.
        
        Creating and sending newsletters and email marketing campaigns to subscribers.
        
        Conducting website updates and maintenance tasks as needed.
        
        Managing Google Analytics and other marketing performance tracking tools to measure success of content and social media campaigns.
      `,
    },
  },
];
