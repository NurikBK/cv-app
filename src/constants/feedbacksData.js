export const feedbackData = [
  {
    feedback: `I recently worked with Nurzhan on a web development project, and I was thoroughly impressed with their skills and knowledge. They were able to take my design and turn it into a fully functional and responsive website using React and Material UI. Their attention to detail and ability to problem-solve made the project run smoothly, and the end result exceeded my expectations. I highly recommend Nurzhan for any web development projects.`,
    reporter: {
      photoUrl: 'https://i.pravatar.cc/150?img=3',
      name: 'John Doe',
      citeUrl: 'https://www.citeexample.com',
    },
  },
  {
    feedback: `I had the pleasure of working with Nurzhan on a complex web development project that involved NextJS and SCSS. They were able to navigate through the challenges of the project with ease, and their ability to collaborate with the rest of the team made the project a success. Their understanding of the latest web development technologies and their attention to detail resulted in a website that is not only visually stunning but also highly functional. I highly recommend Nurzhan for any web development projects`,
    reporter: {
      photoUrl: 'https://i.pravatar.cc/150?img=10',
      name: 'Jane Rober',
      citeUrl: 'https://www.citeexample.com',
    },
  },
];
