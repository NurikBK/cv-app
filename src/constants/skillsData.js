export const skillsData = [
  { id: 1, name: 'CSS', range: '60' },
  { id: 2, name: 'HMTL', range: '80' },
];
