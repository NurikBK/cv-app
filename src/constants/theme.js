export const theme = {
  colors: {
    accent: '#26c17e',
    secondary: '#fff',
    dark: '#667081',
    darkBg: '#222935',
  },
};
