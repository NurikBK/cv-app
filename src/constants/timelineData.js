export const timelineData = [
  {
    date: '2011',
    title: 'Dalian University Of Foreign Languages',
    text: ` I attended Dalian University of Foreign Languages (DLUFL) in Liaoning, China, where I pursued a Bachelor of Arts in Chinese Literature. Throughout my four years of study, I was exposed to a wide range of literature, from classic works to contemporary pieces. In addition to studying literature in Chinese, I also had the opportunity to learn about Chinese literature and culture through elective courses. Outside of the classroom, I participated in various extracurricular activities, including a literature club. These experiences broadened my perspective and deepened my appreciation for the power of language and storytelling.`,
  },
  {
    date: '2012',
    title: 'Master of Business Administration',
    text: `I earned my Master of Business Administration (MBA) degree from Alamty University of Bussnes in the Kazakhstan. The program provided me with a comprehensive understanding of business concepts, such as marketing, finance, accounting, and operations management. I also had the opportunity to specialize in entrepreneurship and innovation, which allowed me to develop skills in identifying and pursuing new business opportunities. Throughout the program, I worked on various group projects, which improved my collaboration and leadership skills. The MBA program also provided me with a strong professional network that I can tap into for career opportunities.`,
  },
  {
    date: '2023',
    title: 'EPAM',
    text: `EPAM Systems, Inc. is an American company that specializes in software engineering services, digital platform engineering, and digital product design, operating out of Newtown, Pennsylvania. I finished the EPAM web developer course, where I learned about HTML, CSS, JS, and React 'During my studies, I gained a deep understanding of computer architecture, algorithms, data structures, programming languages such as HTML, CSS, JavaSctripts and React. I also had the opportunity to work on various projects, including a web app for tracking fitness goals and a web application for online shopping. These experiences honed my skills in problem-solving, collaboration, and communication. Overall, my degree program provided me with a strong foundation in web development and prepared me for a career in web development.`,
  },
];
