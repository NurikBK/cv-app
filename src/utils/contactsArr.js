import { faPhone, faEnvelope } from '@fortawesome/free-solid-svg-icons';
import {
  faTwitter,
  faFacebookF,
  faSkype,
} from '@fortawesome/free-brands-svg-icons';

export const contacts = [
  {
    icon: faPhone,
    title: '500 342 242',
    info: '',
    link: 'tel:+500 342 242',
  },
  {
    icon: faEnvelope,
    title: 'office@kamsolutions.pl',
    info: '',
    link: 'mailto:office@kamsolutions.pl',
  },
  {
    icon: faTwitter,
    title: 'Twitter',
    info: 'www.twitter.com/wordpress',
    link: 'https://twitter.com/wordpress',
  },
  {
    icon: faFacebookF,
    title: 'Facebook',
    info: 'www.facebook.com/facebook',
    link: 'https://www.facebook.com/facebook',
  },
  {
    icon: faSkype,
    title: 'Skype',
    info: 'kamsolutions.pl',
    link: 'skype:kamsolutions.pl',
  },
];
