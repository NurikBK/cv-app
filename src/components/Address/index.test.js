import React from 'react';
import { render, screen } from '@testing-library/react';
import { ThemeProvider } from 'styled-components';
import { theme } from '../../constants/theme';
import Address from '.';
import { contacts } from '../../constants/contactsArr';

describe('Article component', () => {
  it('renders Address titles', () => {
    render(
      <ThemeProvider theme={theme}>
        <Address />
      </ThemeProvider>
    );
    const contactsTitle = contacts.map((contact, index) =>
      screen.getByText(contact.title)
    );
    contactsTitle.forEach((el) => expect(el).toBeInTheDocument());
  });
});
