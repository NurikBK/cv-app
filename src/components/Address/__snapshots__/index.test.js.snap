// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article component shows out of cheese error message 1`] = `
<address
  class="sc-bgqQcB jFOXYN address"
>
  <div
    class="wrapper"
  >
    <i>
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-phone "
        data-icon="phone"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M164.9 24.6c-7.7-18.6-28-28.5-47.4-23.2l-88 24C12.1 30.2 0 46 0 64C0 311.4 200.6 512 448 512c18 0 33.8-12.1 38.6-29.5l24-88c5.3-19.4-4.6-39.7-23.2-47.4l-96-40c-16.3-6.8-35.2-2.1-46.3 11.6L304.7 368C234.3 334.7 177.3 277.7 144 207.3L193.3 167c13.7-11.2 18.4-30 11.6-46.3l-40-96z"
          fill="currentColor"
        />
      </svg>
    </i>
    <div
      class="info"
    >
      <a
        href="tel:+7 708 693 23 41"
      >
        +7 708 693 23 41
      </a>
      <br />
    </div>
  </div>
  <div
    class="wrapper"
  >
    <i>
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-envelope "
        data-icon="envelope"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M48 64C21.5 64 0 85.5 0 112c0 15.1 7.1 29.3 19.2 38.4L236.8 313.6c11.4 8.5 27 8.5 38.4 0L492.8 150.4c12.1-9.1 19.2-23.3 19.2-38.4c0-26.5-21.5-48-48-48H48zM0 176V384c0 35.3 28.7 64 64 64H448c35.3 0 64-28.7 64-64V176L294.4 339.2c-22.8 17.1-54 17.1-76.8 0L0 176z"
          fill="currentColor"
        />
      </svg>
    </i>
    <div
      class="info"
    >
      <a
        href="mailto:bizhannk@gmail.com"
      >
        bizhannk@gmail.com
      </a>
      <br />
    </div>
  </div>
  <div
    class="wrapper"
  >
    <i>
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-twitter "
        data-icon="twitter"
        data-prefix="fab"
        focusable="false"
        role="img"
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
          fill="currentColor"
        />
      </svg>
    </i>
    <div
      class="info"
    >
      <a
        href="https://twitter.com/wordpress"
      >
        Twitter
      </a>
      <br />
      <small>
        @Nurik
      </small>
    </div>
  </div>
  <div
    class="wrapper"
  >
    <i>
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-facebook-f "
        data-icon="facebook-f"
        data-prefix="fab"
        focusable="false"
        role="img"
        viewBox="0 0 320 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
          fill="currentColor"
        />
      </svg>
    </i>
    <div
      class="info"
    >
      <a
        href="https://www.facebook.com/facebook"
      >
        Facebook
      </a>
      <br />
      <small>
        Nurzhan Bizhan
      </small>
    </div>
  </div>
  <div
    class="wrapper"
  >
    <i>
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-skype "
        data-icon="skype"
        data-prefix="fab"
        focusable="false"
        role="img"
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M424.7 299.8c2.9-14 4.7-28.9 4.7-43.8 0-113.5-91.9-205.3-205.3-205.3-14.9 0-29.7 1.7-43.8 4.7C161.3 40.7 137.7 32 112 32 50.2 32 0 82.2 0 144c0 25.7 8.7 49.3 23.3 68.2-2.9 14-4.7 28.9-4.7 43.8 0 113.5 91.9 205.3 205.3 205.3 14.9 0 29.7-1.7 43.8-4.7 19 14.6 42.6 23.3 68.2 23.3 61.8 0 112-50.2 112-112 .1-25.6-8.6-49.2-23.2-68.1zm-194.6 91.5c-65.6 0-120.5-29.2-120.5-65 0-16 9-30.6 29.5-30.6 31.2 0 34.1 44.9 88.1 44.9 25.7 0 42.3-11.4 42.3-26.3 0-18.7-16-21.6-42-28-62.5-15.4-117.8-22-117.8-87.2 0-59.2 58.6-81.1 109.1-81.1 55.1 0 110.8 21.9 110.8 55.4 0 16.9-11.4 31.8-30.3 31.8-28.3 0-29.2-33.5-75-33.5-25.7 0-42 7-42 22.5 0 19.8 20.8 21.8 69.1 33 41.4 9.3 90.7 26.8 90.7 77.6 0 59.1-57.1 86.5-112 86.5z"
          fill="currentColor"
        />
      </svg>
    </i>
    <div
      class="info"
    >
      <a
        href="skype:nurik"
      >
        Skype
      </a>
      <br />
      <small>
        Nurik
      </small>
    </div>
  </div>
</address>
`;
